components/webapps/operator-onboard/src/main/java/com/wso2telco/dep/service/OperatorService.java

```diff
@@ -36,4 +39,33 @@ public Response addOperator(String jsonBody) throws Exception{
 		
 		return null;	
 	}
+
+	public void fooMethod(String abc) {
+
+		try {
+			Thread.sleep(1000);
+			FileInputStream file = new FileInputStream("hard_coded_path");
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Close this "FileInputStream". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS2095)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "file". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "file" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1481)